src/assets/img/Logo.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="204" height="32" viewBox="0 0 204 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please, create an react component. I mean create a folder called `Icons` inside of `components` folder and then create a component called `Logo.jsx` and render svg inside

Reply:
I have done it, but I'm not sure that it's right